docs/api.md

```diff
@@ -552,6 +552,79 @@ Output content of the table to a `pandas.DataFrame`.
     
     Maximum number of rows to return. Defaults to -1, i.e. all.
 
+## API Client
+
+
+### `sematic.client.block_on_run`
+
+
+Block on the run with the given id until it is in a terminal state.
+
+Terminal states include successful completion, failure and cancellation.
+Only successful completion returns without error, other terminal states
+result in `RuntimeError` being raised.
+
+#### Parameters
+
+- `run_id`: str
+
+    The id of the run to block on.
+
+- `delay_seconds`: float
+
+    The number of seconds between polling for updates to the run's status.
+
+- `max_wait_seconds`: float
```

Comment:
```suggestion
- `max_wait_seconds`: Optional[float]
```

Reply:
It is optional to pass it in, but the type is still float; the default value is 5.0. I will note this in the doc.

Reply:
fixed